android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -305,6 +305,16 @@ public ThemedReactContext getThemedReactContext() {
     return (ThemedReactContext) this.getContext();
   }
 
+  /**
+   * A helper to get react tag id by given WebView
+   */
+  public static int getId(WebView webView) {
+    if (webView.getParent() == null) {
+      return -1;
+    }
```

Comment:
I think it would be helpful to have a brief comment explaining how this is relying on the assumption that the WebView is inside a Wrapper

Reply:
thanks for the review. follow up in the commit: https://github.com/react-native-webview/react-native-webview/pull/2874/commits/893ee82fbbb77e34a0a6a3e8c150cbf16d166810